libnavigation-core/build.gradle

```diff
@@ -20,8 +20,8 @@ android {
 }
 
 dependencies {
-    implementation(project(':annotations'))
-    kapt(project(':annotations-processor'))
+    implementation dependenciesList.mapboxAnnotations
```

Comment:
@LukasPaczos does this need to be `implementation`? Saw it was changed here https://github.com/mapbox/mapbox-navigation-android/pull/2268/files#diff-226b34e1fd3bc1d22b5f62b83ae3eba1R23

Reply:
@LukasPaczos I've tested locally with `compileOnly` and run the tests, all them ✅ 🤔

Reply:
The JVM tests will pass, however, when running the code on an Android device you'll run into `ClassNotFoundException`. That's because we are using module type enums in the lookup code, therefore we need the artifact in the runtime as well.

Reply:
Ohh I see, to have access to `MapboxNavigationModuleType` and `MapboxAnnotationConstants` 👍 Thanks for clarifying @LukasPaczos